src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -106,9 +106,9 @@ public FSFileInfo copyFile(Dataset dataset,
             // so I changed exported the timeouts to application.properties to allow for tuning
             // and I am changing this to copy chunks.
             CopyWriter writer = sourceBlob.copyTo(BlobId.of(bucketResource.getName(), targetPath));
-            while (!writer.isDone()) {
```

Comment:
Is this change necessary for the gradle change?

Reply:
No it's not supposed be a part of this PR. I meant to push this commit to another branch. Thanks for catching it!